### ISSUE TITLE
Coverity: Fix issues with udiskslinuxblock.c

### ISSUE THREAD

**Issue by tasleson** [storaged-project/udisks#295]
```
Error: CHECKED_RETURN (CWE-252): [#def27]
udisks-2.7.1/src/udiskslinuxblock.c:140: check_return: Calling "g_file_get_contents" without checking return value (as is done elsewhere 12 out of 13 times).
udisks-2.7.1/src/udiskscrypttabmonitor.c:367: example_checked: Example 1: "g_file_get_contents("/etc/crypttab", &contents, NULL, &error)" has its value checked in "g_file_get_contents("/etc/crypttab", &contents, NULL, &error)".
udisks-2.7.1/src/udiskslinuxblock.c:1638: example_checked: Example 2: "g_file_get_contents("/etc/crypttab", &contents, NULL, error)" has its value checked in "g_file_get_contents("/etc/crypttab", &contents, NULL, error)".
udisks-2.7.1/src/udiskslinuxdriveata.c:561: example_checked: Example 3: "g_file_get_contents(simulate_path, &blob, &blob_len, error)" has its value checked in "g_file_get_contents(simulate_path, &blob, &blob_len, error)".
udisks-2.7.1/src/udiskslinuxencrypted.c:201: example_checked: Example 4: "g_file_get_contents(passphrase_path, out_passphrase, NULL, error)" has its value checked in "g_file_get_contents(passphrase_path, out_passphrase, NULL, error)".
udisks-2.7.1/src/udiskslinuxfilesystem.c:212: example_checked: Example 5: "g_file_get_contents(filesystems_file, &filesystems, NULL, &error)" has its value checked in "g_file_get_contents(filesystems_file, &filesystems, NULL, &error)".
#  138|     value = NULL;
#  139|     /* don't care about errors */
#  140|->   g_file_get_contents (filename,
#  141|                          &value,
#  142|                          NULL,

Error: DEADCODE (CWE-561): [#def28]
udisks-2.7.1/src/udiskslinuxblock.c:2717: assignment: Assigning: "label" = "NULL".
udisks-2.7.1/src/udiskslinuxblock.c:2920: null: At condition "label != NULL", the value of "label" must be "NULL".
udisks-2.7.1/src/udiskslinuxblock.c:2920: dead_error_condition: The condition "label != NULL" cannot be true.
udisks-2.7.1/src/udiskslinuxblock.c:2920: dead_error_line: Execution cannot reach the expression "label" inside this statement: "command = subst_str_and_esc...".
# 2918|         tmp = subst_str_and_escape (fs_info->command_validate_create_fs, "$DEVICE",
# 2919|                                     udisks_block_get_device (block));
# 2920|->       command = subst_str_and_escape (tmp, "$LABEL", label != NULL ? label : "");
# 2921|         g_free (tmp);
# 2922|   
```

**Comment by vojtechtrefny** [comment]
#403 fixes the second issue. I think we don't need to fix the first one. `g_file_get_contents` sets value to `NULL` if it fails to read the file and we do check it.

**Comment by vojtechtrefny** [comment]
Fixed in #528
